### DISCUSSION TITLE
Create automatically daily journal note

### DISCUSSION THREAD

**Discussion by davidszerman** [TriliumNext/Trilium#3255] [Q&A] [answered]
Hello,

I would like the "today" note to be created automatically everyday.

I think it is related to this wiki page : https://github.com/zadam/trilium/wiki/Events

How should it work ?

Add #run="daily" or #run="???Startup" 
to the Today note ? (not sure, it seems to create the button only...) to another note with a code ?

Thanks

**Comment by zadam** [comment, marked as answer]
I would put `#run=hourly #run=backendStartup` with JS backend note:

```javascript
api.getTodayNote(); // will create if doesn't exist
```

* `#run=daily` runs at unspecified time during the day, so may be too late. `hourly` should be better.
* `#run=backendStartup` will make sure immediately after startup that today's note exists

**Reply by davidszerman** [reply]
It works like a charm ! You are a genius ! (and I am going to open another ticket as I have got a lot of ideas while using my trilium instance   :-D)
